src/Hng.Application/Features/Products/Commands/AddProductsCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Hng.Application.Features.Products.Dtos;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Features.Products.Commands
+{
+    public class AddProductsCommand : IRequest<ProductsDto>
+    {
+        public AddProductsCommand(string userId, List<ProductCreationDto> productCreation)
+        {
```

Comment:
This command is for adding Product, it has no business taking userId as a parameter/argument. If you want to assign a user to a product. Do it in the handler.